tests/execute/local_subprocess/test_local_subprocess.py

```diff
@@ -275,3 +276,23 @@ def test_local_subprocess_tty(monkeypatch: MonkeyPatch, mocker: MockerFixture, t
         "stdin": False,
         "terminal": [100, 100],
     }
+
+
+@pytest.mark.parametrize("mode", ["stem", "full", "stem-pattern", "full-pattern", "all"])
+def test_allow_list_external_ok(fake_exe_on_path: Path, mode: str) -> None:
+    exe = f"{fake_exe_on_path}{'.EXE' if sys.platform == 'win32' else ''}"
+    allow = exe if "full" in mode else fake_exe_on_path.stem
+    allow = f"{allow[:-2]}*" if "pattern" in mode else allow
+    allow = "*" if mode == "all" else allow
```

Comment:
I would prefer to use `allow` as parameter for this test, but you possible had something in mind :-)

Reply:
You don't know the value of fake_exe_on_path upfront to be able to do that 😊